QKSMS/src/main/java/com/moez/QKSMS/ui/conversationlist/ConversationListFragment.java

```diff
@@ -62,6 +62,8 @@
     //@Bind(R.id.recording_text) QKTextView recording_text;
 
     private SidebandDBSource sideDb;
+    public static int sTagUpdateFlag;
```

Comment:
ConversationListFragment will be initialized before we click a conversation and go to MessageListActivity. I just used static flag so that I can set it from MessageListFragment whenever I want ConversationListFragment to update the listview because updating it every time we return form MessageListActivity would be useless most of the time. 
We can also use startActivityForResult to save some flag indicating the listview is updated and call notifyDatasetChanged in onActivityResult or we can use sharedPreferences. Are there better ways of doing this, What do you suggest?